requirements_test_min.txt

```
@@ -8,3 +8,4 @@ pytest-benchmark~=4.0
 pytest-timeout~=2.1
 towncrier~=23.6
 requests
+setuptools
```

That makes sense, what about fixing the setuptools version to the old code that was live during #3152. That's october 2019, so :
```suggestion
setuptools==41.6.0
```

---

I guess what I don't love about that is that you'd need separate environments for testing and building. Could cause confusion.

---

Do we ever build ? I  was at some point but now the release is done by a github action and I never build locally (or tox build locally?)

---

Fair point!